### PR TITLE
Added additional information on Upgrade Assistant usage

### PR DESCRIPTION
**Pull request by tomek-i**
The rule needs some rework, it feels incomplete and a User with little experience would get lost after creating the side-by-side project. I will elaborate in a separate PR more about what to do with YARP and where to go next.

This PR add some more information about what to do next once the side-by-side project has been created and setup.
It touches on the summary page, the end point explorer and talks about the upgrade functionality that Upgrade Assistant offers.

### CODE REVIEW

File: rules/dotnet-upgrade-for-complex-projects/rule.md [hunk 1]
```diff
@@ -1,169 +1,203 @@
...
+
+Outlined below are rules designed to assist in the project upgrade process during migration. Please note that the applicability of certain rules may vary based on individual project requirements.
+* [Do you know how to migrate from System.Web to modern alternatives?](https://www.ssw.com.au/rules/migrate-from-system-web-to-modern-alternatives/)
+* [Do you know how to migrate from EDMX to EF Core?](https://www.ssw.com.au/rules/migrate-from-edmx-to-ef-core/)
+
+# Web application
+
+There are several ways to migrate project from ASP.NET to ASP.NET Core. We strongly recommend using the Strangler Fig pattern to incrementally migrate your project with [YARP](https://microsoft.github.io/reverse-proxy/).
+
+### Create side-by-side incremental project with [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant)
```
Comment (ncn-ssw) [inline rules/dotnet-upgrade-for-complex-projects/rule.md:99]: Now that we have the other Upgrade Assistant section below, do we still require this section?

File: rules/dotnet-upgrade-for-complex-projects/rule.md [hunk 2]
```diff
@@ -1,169 +1,203 @@
...
+
+From here you can explore your endpoints through the `Endpoint explorer`, which will also indicate what endpoints have already been migrated and which ones are still outstanding.
+The chain icon ![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/d89d7150-e0b3-4947-abeb-0e1f865ab6f8) indicates that this endpoint has been migrated and is linked between
+the controller in the old project and the controller in the Yarp proxy project.
+
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/15377711-45a9-41dd-88b5-c555b64e6a87)
+
+**Figure: Endpoint Explorer showing the endoints between the old .NET Framework project and the new .NET Core project.**
+
+Use the `Upgrade` functionality for example the `Upgrade Controller` to let Upgrade Assistant help you with the migration by analyzing and applying automatic code transformations to speed up the process.
```
Comment (ncn-ssw) [inline rules/dotnet-upgrade-for-complex-projects/rule.md:181]: I think this sentence could be improved. I suggest skipping the example reference and rewording this sentence to something like:

> "Use the `Upgrade` functionality to apply automatic code transformations and speed up the migration process."

File: rules/dotnet-upgrade-for-complex-projects/rule.md (hunk 1 above)
Comment (tomek-i) [reply rules/dotnet-upgrade-for-complex-projects/rule.md:100]: As discussed earlier today, this is one of the sections that would benefit of some more details and context.
It owuld be good to have a few steps on how to create the side-by-side, and then some explenation what to do next once that is done eg. setup the routes on your frontend to point to the proxy
and configure the proxy to add a catchall to send everything to the original project.

And then it continues with my edits.

File: rules/dotnet-upgrade-for-complex-projects/rule.md (hunk 2 above)
Comment (tomek-i) [reply rules/dotnet-upgrade-for-complex-projects/rule.md]: applied chages.

**Review by ncn-ssw — APPROVED** [review]

**Pull request merged**